site/content/integrate/apps/quickstart/quick-start-jwt/_index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "JWT in Go"
+heading: "Use JWT in a Mattermost App in Go"
+description: "This quick start guide will walk you through the basics of using JWT in a Mattermost App in Go."
+weight: 30
+---
+
+This quick start guide will walk you through the basics of using [JSON Web Tokens (JWT)](https://jwt.io/) in a Mattermost App in Go. In this guide you will review an App that:
+
+- Contains a `manifest.json`, declares itself an HTTP application that acts as a bot, uses JWT, and attaches to locations in the Mattermost interface.
+- Attaches the `send` function to a `/hello-jwt` command and wraps it with a function to authenticate JWT.
+
+## Prerequisites
+
+Before you can start with your App, you first need to set up your environment by following the [developer setup guide]({{< ref "/integrate/apps/quickstart" >}}).
+
+You also need Go v1.16 or later installed. Please follow the [official guide](https://golang.org/doc/install) to install the latest version.
+
+## Download and start the App
+
+In the same [mattermost-app-examples repository](https://github.com/mattermost/mattermost-app-examples) you cloned via the [developer setup guide]({{< ref "/integrate/apps/quickstart" >}}) above, navigate to the [`golang/jwt`](https://github.com/mattermost/mattermost-app-examples/tree/master/golang/jwt) directory and start the Docker container:
+
+```sh
+cd golang/jwt
+docker compose up
+```
+
+You'll see Docker install the Go modules and then the App will come online and print the following message:
+
+```
+Use '/apps install http http://mattermost-apps-golang-jwt:8084/manifest.json' to install the app
+Use "1234" as the app's JWT secret
+```
+
+## Install the App on Mattermost
+
+Next, access your development Mattermost Server at [http://localhost:8065](http://localhost:8065) and use the `/apps install http http://mattermost-apps-golang-jwt:8084/manifest.json` slash command to install the JWT App. Select `Agree to grant the app access to APIs and Locations`, enter the JWT secret (1234), and click `Submit` to finish the installation.
```

Comment:
I personally think we should spell it "Mattermost server", except for when we are talking directly about the product like "the Mattermost Server". 2/5 An instance a Mattermost server doesn't need to be capitalized.

Reply:
I'll gladly go through and standardize these guides if we can establish a reference document, or someone can do the whole repo in one big sweep (might be simpler). I agree in this case it probably doesn't ned to be capitalized, I had it lowercase before.

Reply:
Another great consistency point to clarity! The product docs use a capital "S" when referring to the binary, and a lowercase "s" when referring to the system that's running Mattermost code.

Reply:
@azigler - I'd recommend holding off on an end-to-end sweep until @justinegeffen makes the terminology reference available.